Structure.Rmd

```diff
@@ -0,0 +1,398 @@
+# Package structure {#package-structure}
+
+```{r, include = FALSE}
+source("common.R")
+```
+
+This chapter will start you on the road to package development by converting the implicit knowledge you've gained from *using* R packages into the explicit knowledge needed to *create and modify* them. You'll learn about the various states a package can be in and the difference between a package and library (and why you should care). Finally, you'll get a blueprint for creating your own packages *de novo*.
+
+## What is a package? {#package}
+
+When you create or modify a package, you work on its "source code" or "source files". You interact with the in-development package in its *source form*. This is not the package form you are most familiar with from day-to-day usage. Package development workflows make much more sense if you understand the five states an R package can be in:
```

Comment:
If you're attempting to make "source form" a definition, I'd bold it, and restructure the sentence to make it more clear you're defining this as a new term